testsuite/generate-format/tests/staged/api.yaml

```diff
@@ -488,14 +483,10 @@ TransactionPayload:
         NEWTYPE:
           TYPENAME: Script
     1:
-      ModuleBundle:
-        NEWTYPE:
-          TYPENAME: ModuleBundle
-    2:
       EntryFunction:
         NEWTYPE:
           TYPENAME: EntryFunction
-    3:
+    2:
```

Comment:
Here show's the difference in the BCS encoding, so we need to add them back.  We can remove everywhere it's used, but these numbers associated need to be the same.

Reply:
Thanks, added back!